BinaryTree/dotNet_lab1/MyList.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dotNet_lab1
+{
+    public class MyList<T> : IEnumerable<T>
```

Comment:
It is not a doubly linked list implementation, it is just a wrapper for an array